app/views/coronavirus_local_restrictions/show.html.erb

```diff
@@ -50,4 +50,21 @@
       } %>
     <% end %>
   </div>
+  <% if t("coronavirus_local_restrictions.related_navigation").is_a?(Array) %>
+    <div class="govuk-grid-column-one-third">
+      <%= render 'govuk_publishing_components/components/related_navigation', {
+        "content_item": {
+          "links" => {
+            "ordered_related_items" => t("coronavirus_local_restrictions.related_navigation").map do | item |
```

Comment:
Are there any legs in moving this logic into the controller? Just thinking it could clean up the view a bit

Reply:
We should ideally move this into a presenter if possible

Reply:
good shout. That'd be great if we can :)